scala/finatra/build.sbt

```
@@ -4,6 +4,6 @@ scalaVersion := "2.13.5"
 lazy val root = (project in file("."))
   .settings(
     libraryDependencies ++= Seq(
-      "com.twitter" %% "finatra-http" % "[21.1,21.2)"
+      "com.twitter" %% "finatra-http" % "[21.2,21.3)"
```

should it not be 
```suggestion
      "com.twitter" %% "finatra-http" % "[21.3,21.4)"
```
? @whiplash